apps/server/src/modules/server/controller/admin-api-server.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller()
+export class AdminApiServerController {
+	@Get()
+	getHello(): string {
```

Comment:
ping() with return { timestamp } is a more common way or?
For what it is used?

Reply:
I think it can be removed I was doing the same as in Server module where controller is specified and tested

Reply:
done